crates/e2e/src/setup/services.rs

```diff
@@ -91,7 +91,7 @@ impl<'a> Services<'a> {
             "--auction-update-interval=1s".to_string(),
             format!("--ethflow-contract={:?}", self.contracts.ethflow.address()),
             "--skip-event-sync=true".to_string(),
-            "--solve-deadline=2s".to_string(),
+            "--solve-deadline=11s".to_string(),
```

Comment:
I assume this isn't messing with our tests since baseline/naive return early and never use the full time? Otherwise we could also make it configurable.

Reply:
I had the same though and wanted to confirm by running CI once before commenting. Indeed it looks like this would be an issue for us. The CI time went from `~5m` to `~10m`.
The reason is that even if the baseline solver is able to compute a solution way faster the driver delays the submission of the solution until shortly before the deadline in case the solution would start to revert at some point.

Reply:
Ah right, I also figured it was a max time. But I checked back on the logic where you implemented this and, as you mentioned, I see the driver then delays until the end of the duration.

I would suggest to add a `solve_deadline: Option<Duration>` argument to `start_autopilot`, where it defaults to 2s if `None`. I feel like a separate function with only one difference is overkill but it would make the diff smaller. What do you think?

Reply:
In the case that you agree, I've committed this change now. Happy to change the approach if you want though.